fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCQueryBuilder.java

```diff
@@ -936,9 +995,23 @@ protected SqlQueryData processInclusionCriteria(QueryParameter queryParm) throws
                 // (pX.PARAMETER_NAME_ID = x AND
                 this.populateNameIdSubSegment(whereClauseSegment, currentParm.getCode(), PARAMETER_TABLE_ALIAS);
                 whereClauseSegment.append(AND);
-                // Build this piece: pX.str_value = search-attribute-value
-                whereClauseSegment.append(PARAMETER_TABLE_ALIAS + DOT).append(STR_VALUE).append(operator)
+                
+                // split the resource type name out (since issue #1366)
+                String resourceTypeName = null;
+                String[] parts = currentParmValue.split("/");
+                if (parts.length == 2) {
+                    resourceTypeName = parts[0];
+                    currentParmValue = parts[1];
+                }
```

Comment:
Do we actually care if this is length 2? when would it not be 2? should it be <= to consider _history?

Reply:
There aren't any unit tests for _history so I guess it's not currently supported or even handled.  Lee has an issue #1273 open to support it properly (which is now possible). I think this will need to be addressed as part of that